api/src/wfl/server.clj

```diff
@@ -80,25 +80,28 @@
 
 (defn ^:private do-update!
   "Update `_workload` in a database transaction. "
-  [{:keys [id uuid labels] :as _workload}]
+  [{:keys [id] :as _workload}]
   (jdbc/with-db-transaction [tx (postgres/wfl-db-config)]
     (let [workload (workloads/load-workload-for-id tx id)]
       (try
         (workloads/update-workload! tx workload)
         (catch UserException e
           (log/warning "Error updating workload"
-                       :exception e :labels labels :workload uuid)
+                       :workload  (workloads/to-log workload)
+                       :exception e)
           (slack/notify-watchers workload (.getMessage e)))))))
 
 (defn ^:private try-update
   "Try to update the workflows in `workload` with a backstop."
-  [{:keys [uuid labels] :as workload}]
+  [workload]
   (try
-    (log/info "Updating workload" :labels labels :workload uuid)
+    (log/info "Updating workload"
+              :workload (workloads/to-log workload))
     (do-update! workload)
     (catch Throwable t
       (log/error "Failed to update workload"
```

Comment:
An example log emitted for a staged workload whose update loop has thrown.  (The passed-in `workload` is a DB record in this case, not a loaded workload object.)

```{"message":{"column":7,"namespace":"wfl.server","result":"Failed to update workload","throwable”:<…throwable…>,"workload":{"uuid":"e66439f4-cb6d-4d8b-93e8-03a1c309dcd6","executor":"TerraExecutor","labels":["hornet:test","project:(Test) rfricke/GH-1566-workloadinfologs"],"source":"TerraDataRepoSource","sink":"TerraDataRepoSink"}},"severity":"ERROR","logging.googleapis.com/sourceLocation":{"file":"wfl/server.clj","function":"\"Failed to update workload\"","line":102},"time":"2022-02-17T18:35:34.069463Z"}```

Reply:
Mostly, I think the `more` keys following the `expression` should almost always be something like `::log/labels` introducing a map containing a workload UUID or dataset name -- and then only when that isn't evident in the `expression`.